### PR TITLE
missing scope with last php backguar compatibility

### PR DESCRIPTION
**Pull request by mckaygerhard**
cannot call selft if there are in no context due object class name..
so now we can call the pulgin reuse from others

### CODE REVIEW

**Comment by clonemeagain** [comment]
How did I miss this one? 

Sorry mate, have been stupid busy.  Looks like it should be `self::method` rather than `$self::method`.. hmm, odd.

File: class.AttachmentPreviewPlugin.php [hunk 1]
```diff
@@ -120,18 +120,20 @@ function bootstrap() {
...
 // Provide a connection point to the attachments.wrapper
         Signal::connect(self::signal_id, function ($object, $data) {
-            $self::$instance->debug_log("Received connection from %s", get_class($object));
+            if (version_compare(PHP_VERSION, '5.4.0') >= 0)
+                $self::$instance->debug_log("Received connection from %s", get_class($object));
 // 
 // Assumes you want to edit the DOM with your structures, and that you've read the docs.
 // Just save them here until the page is done rendering, then we'll make all these changes at once:
-            self::$foreign_elements[get_class($object)] = $data;
+            AttachmentPreviewPlugin::$foreign_elements[get_class($object)] = $data;
```
Comment (mckaygerhard) [inline class.AttachmentPreviewPlugin.php:128]: its selft of course, you can edit the pull due has the "edit by authors" flag enabled

**Comment by mckaygerhard** [comment]
yeah, yes... this has the flag "allow edits from mantainers" so you can edit or trash the pull .. the next week i'll return to osticket developmnet

**Comment by clonemeagain** [comment]
Patched in another branch, cheers for spotting that mate!

**Comment by mckaygerhard** [comment]
hi long time without see you here! thanks for pathing!

**Pull request closed without merging**